### ISSUE TITLE
Stable Audio not working

### ISSUE THREAD

**Issue by mykeehu** [rsxdalv/tts-generation-webui#418]
I updated TTS today and when I tried to generate music on the Stable Audio page, I got an error:

```
Prompt: pleasant, soft melody for pensive, romantic background music
Traceback (most recent call last):
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\gradio\queueing.py", line 622, in process_events
    response = await route_utils.call_process_api(
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\gradio\route_utils.py", line 323, in call_process_api
    output = await app.get_blocks().process_api(
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\gradio\blocks.py", line 2014, in process_api
    result = await self.call_function(
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\gradio\blocks.py", line 1567, in call_function
    prediction = await anyio.to_thread.run_sync(  # type: ignore
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\anyio\to_thread.py", line 56, in run_sync
    return await get_async_backend().run_sync_in_worker_thread(
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\anyio\_backends\_asyncio.py", line 2177, in run_sync_in_worker_thread
    return await future
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\anyio\_backends\_asyncio.py", line 859, in run
    result = context.run(func, *args)
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\gradio\utils.py", line 846, in wrapper
    response = f(*args, **kwargs)
  File "I:\tts-generation-webui\tts-generation-webui-main\tts_webui\stable_audio\stable_audio_tab.py", line 48, in generate_cond_lazy
    return generate_cond(
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\stable_audio_tools\interface\gradio.py", line 107, in generate_cond
    device = next(model.parameters()).device
AttributeError: 'NoneType' object has no attribute 'parameters'
{'date': '2024-11-12_10-25-30', 'version': '0.0.1', 'prompt': 'pleasant, soft melody for pensive, romantic background music', 'negative_prompt': '', 'seconds_start_slider': 0, 'seconds_total_slider': 60, 'cfg_scale_slider': 7, 'steps_slider': 100, 'preview_every_slider': 0, 'seed_textbox': '2344631576', 'sampler_type_dropdown': 'dpmpp-3m-sde', 'sigma_min_slider': 0.03, 'sigma_max_slider': 500, 'cfg_rescale_slider': 0, 'init_audio_checkbox': False, 'init_audio_input': None, 'init_noise_level_slider': 0.1}
Traceback (most recent call last):
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\gradio\queueing.py", line 622, in process_events
    response = await route_utils.call_process_api(
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\gradio\route_utils.py", line 323, in call_process_api
    output = await app.get_blocks().process_api(
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\gradio\blocks.py", line 2014, in process_api
    result = await self.call_function(
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\gradio\blocks.py", line 1567, in call_function
    prediction = await anyio.to_thread.run_sync(  # type: ignore
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\anyio\to_thread.py", line 56, in run_sync
    return await get_async_backend().run_sync_in_worker_thread(
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\anyio\_backends\_asyncio.py", line 2177, in run_sync_in_worker_thread
    return await future
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\anyio\_backends\_asyncio.py", line 859, in run
    result = context.run(func, *args)
  File "I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\gradio\utils.py", line 846, in wrapper
    response = f(*args, **kwargs)
  File "I:\tts-generation-webui\tts-generation-webui-main\tts_webui\stable_audio\stable_audio_tab.py", line 336, in save_result
    sr, data = audio
TypeError: cannot unpack non-iterable NoneType object
I:\tts-generation-webui\tts-generation-webui-main\installer_files\env\lib\site-packages\gradio\blocks.py:1747: UserWarning: A function (torch_clear_memory) returned too many output values (needed: 0, returned: 1). Ignoring extra values.
    Output components:
        []
    Output values returned:
        [None]
  warnings.warn(
```

What's my problem?

**Comment by rsxdalv** [comment]
Thanks for the report!
I think I can fix this bug. But the workaround should be this - before you click generate, try to load a model by selecting one in the drop-down.

**Comment by rsxdalv** [comment]
I've added a new button for loading models, I realized that the old UI that automatically loads models was not too great because this is a fairly big model and takes some time to load.

Also, now it should just give a straight up error if the model is not loaded. I might add an automatic load functionality but to keep it simple I am considering just a 'model needs to be loaded' error for the time being.
